articles/architecture-scenarios/b2c.md

```diff
@@ -1,108 +1,144 @@
+::: panel Best Practice
+The Auth0 Dashboard in conjunction with the [Delegated Administration Extension](/extensions/delegated-admin/v3) can be used out-of-the-box to provide for user provisioning and deprovisioning as described in this [Auth0 blog post](https://auth0.com/blog/delegated-admin-v2/). If you require more comprehensive deprovisioning functionality, for compliance reasons for example, then we recommend you do so in a later phase.
+::: 
 
-## Extensibility with augmented user profiles
+### Authentication
 
-You may want to enrich user profiles with data obtained from other sources. [Auth0 Rules](/rules) enable you to write small snippets of code that execute during the authentication transaction. This lets you call other services for user information, then add it to the Auth0 user profile.
+[Authentication](/architecture-scenarios/implementation/b2c/authentication) comes next and can be done in parallel with user provisioning. Topics to address at this point include:
 
-## Passwordless login
+* Universal login
+* Username and password authentication
+* Application integration
 
-Sometimes users forget their passwords. [Passwordless login](/connections/passwordless) lets users authenticate with a one-time code sent via SMS or email. This is useful for applications that aren't used very often or primarily used on small mobile devices where it is cumbersome to enter a password.
+### Branding
 
-Auth0 supports several forms of passwordless login. So based on the needs of your users and application, you can choose the ones that fit. Note that using passwordless login may require you to get additional services. For example, a service to send SMS messages.
+[Branding](/architecture-scenarios/implementation/b2c/branding) can be done in parallel with user provisioning and authentication. Topics to address at this point will include:
 
-## Multi-factor authentication
+* Universal Login page customization
+* Naming for your custom domain
+* Change Password page customization
+* Error Page Customization
+* Email template customization
 
-Is your application handling sensitive content? You may want to offer [multi-factor authentication](/mfa) to your users. With malware threats and data breaches, multi-factor authentication is more popular among users.
+From here on you’ll be working towards Milestone 2, and then on to Milestone 3 which will take you to production Go-Live. As you progress through the remaining workstreams and topics you can start to align your Auth0 tenants with your SDLC, and you’ll be steadily and progressively reducing risk as you go. You’ll also have the opportunity to demonstrate further functionality to stakeholders, which will also help you to garner feedback from the rest of the business:
```

Comment:
Without any diagram (see my previous comment), I think the notion of numbered milestones makes little sense. The suggested changes I think will work in both cases (i.e. with or without the diagram) :-)

```suggestion
By this point you will have completed major work required to integrate an application, will have addressed the most significant risk items, and will also be able to provide demonstrable functionality to stakeholders too. From here on in you’ll be working towards production Go-Live. As you progress through the remaining workstreams and topics you can start to align your Auth0 tenants with your SDLC, and you’ll be steadily and progressively reducing risk as you go. You’ll also have the opportunity to demonstrate further functionality to stakeholders, which will also help you to garner feedback from the rest of the business:
```